### PR TITLE
updates-pde

### PR DESCRIPTION
**Pull request by resmaa**
**3.0 Updates**

**1. Added known issue to release-notes - [PPD-359](https://spectrocloud.atlassian.net/browse/PPD-359)**
image

**2. Beehive- System Scope Cluster group at the us-east location.**
image

**3. Output parameter visibility for app tiers** 
image

**4. SandBox Nested=Nested Cluster**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/944#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/04-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-sandbox-clusters.md:19]: @jgautam and @tiwarisumit3 are you two able to share more details on what selected resources are? It's a bit vague so if we can help the reader that'd be ideal.

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:78]: Customers will not have access to this internal link, therefore we should not expose it as it creates bad UX. Is there a public-facing resource we can point to? Otherwise we best remove it,

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
@resmaa I left some comments below. The changes look good but there are some points of concern that we should address.

File: content/docs/00-release-notes.md
Comment (tiwarisumit3) [reply content/docs/00-release-notes.md:78]: While deploying multiple applications on a Sandbox cluster, if one of application deployment gets stuck can cause other applications within the same Sandbox cluster getting stuck.

File: content/docs/04-clusters/04-sandbox-clusters.md
Comment (tiwarisumit3) [reply content/docs/04-clusters/04-sandbox-clusters.md:19]: This is more details on the virtual clusters: 
https://www.vcluster.com/

Sandbox Clusters are Kubernetes clusters that run as nested clusters within an existing cluster (also known as a Host Cluster) and share host cluster resources. By default, Sandbox Clusters will use [k3s](https://github.com/k3s-io/k3s) as virtual Kubernetes cluster, certified Kubernetes distribution designed for production workloads.

Comment (tiwarisumit3) [reply content/docs/04-clusters/04-sandbox-clusters.md:19]: @karl-cardenas-coding - Is it like the definition you want it to be more clear?

File: content/docs/04.5-devx/05-app-profile.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/05-app-profile.md:54]: To view the available output parameters from the below tiers in the code editor, type in ```{{.spectro.}}```

File: content/docs/04-clusters/04-sandbox-clusters.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-sandbox-clusters.md]: That's better @tiwarisumit3 . Can we say, "share host cluster resources such as CPU, memory, and storage"?

Comment (tiwarisumit3) [reply content/docs/04-clusters/04-sandbox-clusters.md]: sure @karl-cardenas-coding , should be fine.

File: content/docs/00-release-notes.md
Comment (tiwarisumit3) [inline content/docs/00-release-notes.md:78]: While deploying multiple applications on a Sandbox cluster, if one of the application get stuck because of some errors it can cause other deployments in the same sandbox cluster to get stuck in provisioning state.

File: content/docs/00-release-notes.md
Comment (amarjeet27) [inline content/docs/00-release-notes.md:78]: Our UI refers to applications and deployments as **apps**, shouldn't we use the same term in documentation, @tiwarisumit3 @resmaa?

What do you think of the following text?

> While deploying multiple apps in a sandbox cluster, if the deployment of one of the apps is blocked due to errors then subsequent apps deployed to the same sandbox cluster might also be stuck in deploying state.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (amarjeet27) [inline content/docs/04.5-devx/03-cluster-groups.md:33]: @resmaa, as far as I recall, the beehive cluster group shows up in lower case on our UI, isn't that the case @tiwarisumit3?

File: content/docs/04-clusters/04-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-sandbox-clusters.md:19]:
```suggestion
Sandbox Clusters are Kubernetes clusters that run as nested clusters within an existing cluster (also known as a Host Cluster) and share the host cluster resources, such as CPU, memory, and storage. By default, Sandbox Clusters will use [k3s](https://github.com/k3s-io/k3s) as virtual Kubernetes cluster, which is a highly available, certified Kubernetes distribution designed for production workloads.
```

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:19]:
```suggestion
Sandbox Clusters are Kubernetes clusters that run as nested clusters within an existing cluster (also known as a Host Cluster) or Host Cluster groups and share the host cluster resources, such as CPU, memory, and storage. By default, Sandbox Clusters will use k3s as virtual Kubernetes cluster, which is a highly available, certified Kubernetes distribution designed for production workloads.
```

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [reply content/docs/00-release-notes.md:78]: @amarjeet27 We should mention the workaround for this issue. Simply mentioning the issue and not providing the solution can frustrate the reader.

Comment (karl-cardenas-coding) [reply content/docs/00-release-notes.md:78]: @amarjeet27 We should mention the workaround for this issue. Simply mentioning the issue and not providing the solution can frustrate the reader.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/03-cluster-groups.md:33]: We could go with this as it is a console value.

```suggestion
  * Palette provides a default cluster group called `beehive` at the system scope in the us-east region. 
```

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | e896756e8c5605b63581409269fa51813f8aea1f | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/944/librarium_build/1586025019962560512) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/00-release-notes.md
Comment (tiwarisumit3) [reply content/docs/00-release-notes.md:78]: @karl-cardenas-coding - good point. We can mention something like:

1. Delete the stuck deployment. 
2. Fix the app with the error.
3. Redeploy the app again.

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/944#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/944#pullrequestreview-1160363706" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:84]: I recommended we remove the second use of bold text and just tell the reader inline that three workarounds are available. 

```suggestion
**Known Issues**

* While deploying multiple apps in a sandbox cluster, if the deployment of one of the apps is blocked due to errors, then subsequent apps deployed to the same sandbox cluster might also be stuck in deploying state.  Apply the following workarounds if you encounter the issue.

  * Delete the stuck App.
  * Fix the App with the error.
  * Redeploy the App again.  
```

**Review by karl-cardenas-coding — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/944#pullrequestreview-1160398549" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/944#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/944#pullrequestreview-1160363706" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | fe3d1497ac7333d8ffed9366b710042eda61f234 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/944/librarium_build/1586062904803725312) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
